File: Backend/api/urls.py
from django.urls import path
from knox import views
from .views import *

urlpatterns = [
    path('', getRoutes),
    path('profiles', getProfiles),
    path('profiles/create', createProfile),
    path('profiles/<int:pid>/update', updateProfile),
    path('profiles/<int:pid>/delete', deleteProfile),
    path('profiles/<int:pid>', getProfile),
    path('register', RegisterAPI.as_view(), name='Register'),
    path('login', LoginAPI.as_view(), name='Login'),
    path('logout', views.LogoutView.as_view(), name='Logout'),
]

File: Frontend/lib/urls.dart
const String _home = "10.0.2.2:8000";

final Uri profileUrl = Uri.http(_home, "profiles");
final Uri loginUrl = Uri.http(_home, "login");
final Uri registerUrl = Uri.http(_home, "register");
final Uri logoutUrl = Uri.http(_home, "logout");

Uri deleteProfileUrl(int id)
{ return Uri.http(_home, "profiles/$id/delete"); }